Tests/MongoSwiftSyncTests/SyncChangeStreamTests.swift

```diff
@@ -297,10 +297,10 @@ final class ChangeStreamSpecTests: MongoSwiftTestCase, FailPointConfigured {
 }
 
 /// Class for spec prose tests and other integration tests associated with change streams.
-final class ChangeStreamTests: MongoSwiftTestCase {
+final class SyncChangeStreamTests: MongoSwiftTestCase {
     /// How long in total a change stream should poll for an event or error before returning.
     /// Used as a default value for `ChangeStream.nextWithTimeout`
-    public static let TIMEOUT: TimeInterval = 10
+    public static let TIMEOUT: TimeInterval = 15
```

Comment:
I bumped this because I got a spurious error testing against latest sharded again. I'm not sure if it was caused by a performance degradation introduced by the changes in this PR or by the same server side change stream performance problems we've seen already. Based on [SERVER-44356](https://jira.mongodb.org/browse/SERVER-44356), it seems  that events may take around 10s to fire in certain circumstances, so I pushed it past 10s just to be safe.